sequencer/src/api/endpoints.rs

```diff
@@ -46,6 +46,42 @@ pub struct NamespaceProofQueryData {
     pub transactions: Vec<Transaction>,
 }
 
+pub(super) fn get_balance<State, Ver>() -> Result<Api<State, merklized_state::Error, Ver>>
+where
+    State: 'static + Send + Sync + ReadState,
+    Ver: 'static + StaticVersionType,
+    <State as ReadState>::State: Send
+        + Sync
+        + MerklizedStateDataSource<SeqTypes, FeeMerkleTree, 256>
+        + MerklizedStateHeightPersistence,
+    //for<'a> <<UniversalMerkleTree<FeeAmount,Sha3Digest,FeeAccount,256,Sha3Node> as hotshot_query_service::merklized_state::MerklizedState<Types, ARITY>>::Commit as TryFrom<&'a TaggedBase64>>::Error: Display,
+{
+    let mut options = merklized_state::Options::default();
+    let extension = toml::from_str(include_str!("../../api/merklized_state.toml"))?;
+    options.extensions.push(extension);
+
+    let mut api =
+        merklized_state::define_api::<State, SeqTypes, FeeMerkleTree, Ver, 256>(&options)?;
+
+    api.get("getfeebalance", move |req, state| {
+        async move {
+            let address = req.string_param("address")?;
+            let height = state.get_last_state_height().await?;
```

Comment:
I think `height` should be a parameter one can specify. But it can default to latest.

Reply:
@sveitser I'll put this in another ticket so we can merge the current version. I changed the api path to "/fee-balance/latest/:address" so that we don't have to change it later when we add the height as parameter as well.